docs/source/api_reference/models.rst

```diff
@@ -403,6 +403,11 @@ MATModel
 .. autoclass:: deepchem.models.torch_models.MATModel
   :members:
 
+NormalizingFlowModel
+--------------------
+
+.. autoclass:: deepchem.models.torch_models.NormalizingFlow
```

Comment:
Can you add the new layer to the docs as well?

Reply:
I think this new layer is already added at `docs/source/api_reference/layers.rst`